shuffle-storage/src/main/java/com/alibaba/flink/shuffle/storage/partition/LocalFileStorageMeta.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.flink.shuffle.storage.partition;
+
+import com.alibaba.flink.shuffle.common.utils.CommonUtils;
+import com.alibaba.flink.shuffle.core.storage.StorageMeta;
+import com.alibaba.flink.shuffle.core.storage.StorageType;
+import com.alibaba.flink.shuffle.storage.utils.IOUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.channels.FileChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/** {@link StorageMeta} for local file system. */
+public class LocalFileStorageMeta extends StorageMeta {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LocalFileStorageMeta.class);
+
+    public LocalFileStorageMeta(String storagePath, StorageType storageType) {
+        super(storagePath, storageType);
+    }
+
+    @Override
+    public long updateUsableStorageSpace() {
+        numUsableSpaceBytes = Paths.get(storagePath).toFile().getUsableSpace();
+        return numUsableSpaceBytes;
+    }
+
+    @Override
+    public void updateStorageHealthStatus() {
+        Path path = new File(storagePath + CommonUtils.randomHexString(32) + ".test").toPath();
```

Comment:
We'd better define this `.test` string as a final static variable to avoid the magic variables in the production code.